test/cases/api.sh

```diff
@@ -470,8 +512,9 @@ function Test_verifyComposeResultGCP() {
   echo "⏱ Waiting for GCP instance to respond to ssh"
   instanceWaitSSH "$HOST"
 
-  # Check if postgres is installed
-  ssh -oStrictHostKeyChecking=no -i "$GCP_SSH_KEY" "$SSH_USER"@"$HOST" rpm -q postgresql
+  # Verify image
+  _ssh="ssh -oStrictHostKeyChecking=no -i $GCP_SSH_KEY $SSH_USER@$HOST"
```

Comment:
Since yo uadded the `echo "✔️ Instance checking"` in the AWS check, might as well add it here :)

Reply:
Thanks @Gundersanne ! Move it into the function to reduce duplicate lines.